template.rb

```diff
@@ -493,4 +493,28 @@ def gsub_file!(path, flag, ...)
   raise StandardError, "the contents of #{path} did not change!" if content == File.binread(path)
 end
 
+def append_to_file!(path, ...)
+  content = File.binread(path)
```

Comment:
suggestion:non-blocking: I can't think of a situation where we would want to use this on a non utf-8 file. If a file can't be parsed as UTF-8 then it's probably an error so `File.read` is probably a bit better.